### PR TITLE
docs: refine cli documentation

### PR DESCRIPTION
**Pull request by ludanxer**
**Summary**

1. Refine cli documentation
2. Update comments in `checkEnv.js`

Will update Chinese doc after review

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Best wishes to all those fighting the coronavirus in China and all over the world**

### CODE REVIEW

File: packages/docs/docs/api/cli.md [hunk 1]
```diff
@@ -1,52 +1,74 @@
...
 # Command-line Interface
 
+There are currently four cli commands in VuePress: [build](#build), [dev](#dev), [eject](#eject) and [info](#info).
+
+If they are not enough for you, you can also create [your own commands](#more-commands).
+
 ## Usage
 
 ```bash
-vuepress <command> targetDir [options]
```
Comment (kefranabg) [inline packages/docs/docs/api/cli.md]: Why removing targetDir?

Comment (ludanxer) [reply packages/docs/docs/api/cli.md]: `vuepress info` command does not require `targetDir `

Comment (meteorlxy) [reply packages/docs/docs/api/cli.md]: Then the other commands miss `targetDir`

Comment (ludanxer) [reply packages/docs/docs/api/cli.md]: I think that's the difference between `vuepress --help` and `vuepress dev --help`?

Comment (newsbielt703) [reply packages/docs/docs/api/cli.md]: Well, some people won't realize that they have to pass `targetDir ` at first glance. In this case, how about rewriting `Usage` in each section?

Comment (ludanxer) [reply packages/docs/docs/api/cli.md]: After reviewing it again, I think I'm not making too much improvement 😅Really sorry about that, will do more research next time. Does it look a bit better now?

Comment (newsbielt703) [reply packages/docs/docs/api/cli.md]: LGTM!

**Comment by newsbielt703** [comment]
The Chinese translation hasn't been finished

**Pull request merged**